openapi.yaml

```diff
@@ -4843,7 +4843,11 @@ components:
           $ref: '#/components/schemas/process_description'
         optional:
           type: boolean
-          description: Determines whether this parameter is optional to be specified.
+          description: >-
+            Determines whether this parameter is optional to be specified even when no default is specified.
+            
+            Clients SHOULD automatically set this parameter to `true`, if a default value is specified.
+            Parsers SHOULD NOT fail, if a default value is specified and this flag is missing.
```

Comment:
Is "parser" a concept we already use somewhere else? This can be just "back-end" I guess? Or you also want to cover validators and such?

Reply:
No, I don't think it's mentioned before. I was referring to our "generic" parsers, but maybe it's really better to say "back-ends".

Reply:
Fixed in dc520faec7dca30da213b0c25a3f2e03ce1b66fd